src/Compilers/CSharp/Portable/Symbols/AnonymousTypes/PublicSymbols/AnonymousManager.TypeOrDelegatePublicSymbol.cs

```diff
@@ -132,12 +133,12 @@ public sealed override ImmutableArray<NamedTypeSymbol> GetTypeMembers()
                 return ImmutableArray<NamedTypeSymbol>.Empty;
             }
 
-            public sealed override ImmutableArray<NamedTypeSymbol> GetTypeMembers(string name)
+            public sealed override ImmutableArray<NamedTypeSymbol> GetTypeMembers(ReadOnlyMemory<char> name)
```

Comment:
a lot of hte change will be this mechanical switch from having NamedTypeSymbol/NamespaceSymbol move over to ROM<char> over string.  

Of note:

1. I didn't do this for VB sicne we don't have traces showing a problem there (And i wanted to scope things small).
2. I only updated GetTypeMembers, not GetMembers.  The latter is not part of the GetTypeByMetadataName codepath, which is the worst offender in all our traces.
3. I didn't expose this publicly, though we may want to.

Reply:
yeah, i'm basically saying: it's intentional that i'm *not* pushing for this to be public.  and we should only drive that based on evidence that tthere is a need.